scripts/manager_version2.lua

```diff
@@ -543,6 +543,16 @@ function migrateChar5(nodeChar)
       local nodeValue = DB.getValue(nodeOld,"text","");
       DB.deleteChild(nodeOld, "text");
       DB.setValue(nodeOld, "text", "formattedtext", nodeValue);
+      local totalCP = DB.getValue(nodeOld, "points", 0);

```

Comment:
These DB updates will need to be broken out into a new DB Version update, in order to migrate characters to the new structure. 

eg.
function migrateChar9(nodeChar)
// Put the character DB changes here.
end